server/src/projects/projects.service.ts

```diff
@@ -116,7 +123,12 @@ export class ProjectsService {
         }, HttpStatus.NOT_FOUND);
       }
 
-      return project;
+      const projectWithContacts = {
+        ...project,
+        contacts: projectApplicantsWithContacts.records.map(applicant => applicant.dcp_applicant_customer_contact),
+      };
```

Comment:
I know we discussed this at length, and I think this is okay for now, but I thought it'd be better to have our model relationships closely match the entity relationships in CRM. So, if CRM has projects -> project_applicant -> contact, then our app should do the same. 

I do think it's probably okay here but could get confusing upon return. Or maybe not. Something to keep an eye on, though, in terms of design...

For example, if we need to associate a contact with a different project, we would need the dcp_projectapplicant entity to do that. Contacts don't have a direct association with projects.

Reply:
Yes that's a good point -- I originally tried to do it as project.projectApplicant.contact but for some reason I was having trouble getting access to the contacts in the frontend through the projectApplicants. Not sure what I was doing wrong, but I ended up using this shortcut as a fix, which probably isn't the most sustainable strategy.